slack-java-client/src/main/java/com/hubspot/slack/client/SlackWebClient.java

```diff
@@ -915,6 +915,20 @@ protected LazyLoadingPage<Result<List<Conversation>, SlackError>, String> getPag
     };
   }
 
+  private List<Conversation> getConversations(ConversationListResponse conversationListResponse, ConversationsListParams params) {
+    List<Conversation> conversations = conversationListResponse.getConversations();
+    if (params.shouldExcludeSharedChannels().orElse(false)) {
+      return filterSharedConversations(conversations);
```

Comment:
I guess the correct name for this method would be `filterOutShareConversations` because we are filtering all the conversations to remove the shared ones.

Reply:
I've been thinking about this naming too.